flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/SortMergeResultPartitionReadSchedulerTest.java

```diff
@@ -192,6 +199,131 @@ public void testOnReadBufferRequestError() throws Exception {
         assertAllResourcesReleased();
     }
 
+    @Test
+    public void testRequestTimeoutIsRefreshedButFailed() throws Exception {
```

Comment:
The timeout refresh is already verified in the bellow test, so I think there is no need to verify this behavior in this test?

Reply:
This method is used to test the case of buffer request timeout. I have removed the code about refreshing timeout and rename it as `testRequestBufferTimeoutAndFailed`.